### PR TITLE
Minor fixes improving UX of taker shutdown

### PR DESCRIPTION
**Pull request by klochowicz**
- reduce log level of heartbeat message
- improve the toast message to clearly state possible action

### CODE REVIEW

File: daemon/src/connection.rs [hunk 1]
```diff
@@ -138,7 +138,7 @@ impl Actor {
 
         match msg {
             wire::MakerToTaker::Heartbeat => {
-                tracing::info!("got the heartbeat");
+                tracing::debug!("received a heartbeat message from maker");
```
Comment (da-kami) [inline daemon/src/connection.rs]:
```suggestion
                tracing::trace!("received a heartbeat message from maker");
```

Let's do trace please. This is not important information and we run on debug by default. It heavily clutters the logs :)

**Review by da-kami — CHANGES_REQUESTED** [review]
Please also change the daemon message `Maker is offline. Shutting down the taker` to something like `Lost connection to maker, shutting down. Please restart the daemon to reconnect`

For now the user will likely see both messages (in UI and daemon) and it would be great if they roughly say the same things.

**Review by thomaseizinger — APPROVED** [review]
ACK modulo Daniel's suggestion

**Review by da-kami — APPROVED** [review]

**Review by bonomat — APPROVED** [review]
👍

**Pull request merged**